### DISCUSSION TITLE
UTM without macos 12+ preconfigured. Why?

### DISCUSSION THREAD

**Discussion by eder-vasquez** [utmapp/UTM#4746] [UTM Help (macOS)] [answered]
I just bought UTM in my AppStore to create a VM with MacOS but It doesn't have it preconfigured. Should I need to do something else to have it? 

 I was waiting for this 
![image](https://user-images.githubusercontent.com/108436353/205361554-30a8c75f-270f-4413-87fb-16e8b6fdfbe5.png)
but I am having this 
![image](https://user-images.githubusercontent.com/108436353/205361645-9afc7927-b449-42ca-923d-904d5b712ce0.png)

Info about my mac
![image](https://user-images.githubusercontent.com/108436353/205362519-9c9863fc-ebd5-4277-8392-10c071dc465a.png)

**Comment by saagarjha** [comment, marked as answer]
This is only supported on Apple silicon.

**Reply by eder-vasquez** [reply]
Thanks @saagarjha
